gstreamer-audio/src/audio_format_info.rs

```diff
@@ -126,7 +126,7 @@ impl AudioFormatInfo {
                 self.0,
                 flags.to_glib(),
                 dest.as_mut_ptr() as *mut _,
-                src.as_ptr() as *mut _,
+                src.as_ptr() as _,
```

Comment:
Is a cast needed at all here now?

Reply:
afaiu yes, because u8 is not &[u8]

Reply:
See https://github.com/sdroege/gstreamer-sys/pull/18/files#r226812757